### PR TITLE
Client basic commands

### LINKED ISSUE #2: As a sales personnel, I can create a profile for a client.

### PR DESCRIPTION
**Pull request by KamiliArsyad**
#2 

Added support for `create` command with multiple optional fields. The previous `add` command is no longer used and has been renamed to the `create` command.

Note:
Details of a person (e.g. email, phone, etc.,) should be grouped somehow instead of dealing with them manually and hardcoding each attribute.

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/MeowCommand.java
Comment (zhyuhan) [inline src/main/java/seedu/address/logic/commands/MeowCommand.java:1]: Should `meow` be part of this commit? 😅

File: src/main/java/seedu/address/logic/parser/AddCommandParser.java
Comment (zhyuhan) [inline src/main/java/seedu/address/logic/parser/AddCommandParser.java:1]: Shall we rename this to `CreateCommandParser`?

File: src/main/java/seedu/address/logic/parser/CliSyntax.java [hunk 1]
```diff
@@ -11,5 +11,8 @@ public class CliSyntax {
     public static final Prefix PREFIX_EMAIL = new Prefix("e/");
     public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
     public static final Prefix PREFIX_TAG = new Prefix("t/");
-
```
Comment (zhyuhan) [inline src/main/java/seedu/address/logic/parser/CliSyntax.java:14]: Are we keeping the `*/` prefix from AB3 or are we going to switch to using flags for fields?

File: src/main/java/seedu/address/model/person/TelegramHandle.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
+package seedu.address.model.person;
+
+/**
+ * Represents a Person's telegram handle in the address book.
+ */
+public class TelegramHandle {
+
+    public static final String MESSAGE_CONSTRAINTS = "Telegram handles should only contain alphanumeric characters "
+            + "and spaces, and it should not be blank";
+    public static final String VALIDATION_REGEX = "[\\p{Alnum}][\\p{Alnum} ]*";
```
Comment (zhyuhan) [inline src/main/java/seedu/address/model/person/TelegramHandle.java]: Is it a good idea to standardise all Telegram handles to start with `@`?

File: src/test/java/seedu/address/logic/LogicManagerTest.java [hunk 3]
```diff
@@ -165,7 +165,7 @@ public void saveAddressBook(ReadOnlyAddressBook addressBook, Path filePath)
         logic = new LogicManager(model, storage);
 
         // Triggers the saveAddressBook method by executing an add command
-        String addCommand = AddCommand.COMMAND_WORD + NAME_DESC_AMY + PHONE_DESC_AMY
+        String addCommand = CreateCommand.COMMAND_WORD + NAME_DESC_AMY + PHONE_DESC_AMY
```
Comment (zhyuhan) [inline src/test/java/seedu/address/logic/LogicManagerTest.java]: nit: Shall we rename `addCommand` to `createCommand`?

File: src/test/java/seedu/address/logic/commands/CreateCommandTest.java [hunk 4]
```diff
@@ -25,46 +25,47 @@
...
+                CreateCommand.MESSAGE_DUPLICATE_PERSON, () -> createCommand.execute(modelStub));
     }
 
     @Test
     public void equals() {
         Person alice = new PersonBuilder().withName("Alice").build();
         Person bob = new PersonBuilder().withName("Bob").build();
-        AddCommand addAliceCommand = new AddCommand(alice);
-        AddCommand addBobCommand = new AddCommand(bob);
+        CreateCommand addAliceCommand = new CreateCommand(alice);
```
Comment (zhyuhan) [inline src/test/java/seedu/address/logic/commands/CreateCommandTest.java]: nit: command variable name

**Review by zhyuhan — COMMENTED** [review]
Looks good! Left a few minor comments.

File: src/main/java/seedu/address/logic/commands/CreateCommand.java [hunk 5]
```diff
@@ -16,34 +20,43 @@
...
+            + PREFIX_DETAILS + "Likes to play games ";
 
-    public static final String MESSAGE_SUCCESS = "New person added: %1$s";
+    public static final String MESSAGE_SUCCESS = "New client created: %1$s";
     public static final String MESSAGE_DUPLICATE_PERSON = "This person already exists in the address book";
 
     private final Person toAdd;
 
     /**
      * Creates an AddCommand to add the specified {@code Person}
```
Comment (MagnificentCreature) [inline src/main/java/seedu/address/logic/commands/CreateCommand.java]: Nit: Should this(the jDocs) say CreateCommand instead?
```
Creates an CreateCommand to add the specified {@code Person}
```

File: src/main/java/seedu/address/model/person/TelegramHandle.java (hunk 2 above)
Comment (MagnificentCreature) [reply src/main/java/seedu/address/model/person/TelegramHandle.java]: Not all Telegram accounts have handles, but I think we can standardise it to be so

**Review by MagnificentCreature — APPROVED** [review]
Some nits but overall looks good

File: src/main/java/seedu/address/logic/parser/CliSyntax.java (hunk 1 above)
Comment (KamiliArsyad) [reply src/main/java/seedu/address/logic/parser/CliSyntax.java:14]: It seems that for now it's more convenient to stick with the previous rules. Later we can change them altogether if we decide to do so

**Comment by KamiliArsyad** [comment]
I have implemented the changes. Thanks @zhyuhan and @MagnificentCreature for reviewing!

**Pull request merged**